tests/README.md

```diff
@@ -0,0 +1,14 @@
+# Algolia Crawler - Configuration Tester
+
+These scripts are reserved for Algolia employees.
+
+## How to test a configuration
+
+Make sure you have Node.js installed, then run the following commands:
+
+```sh
+$ cd tests
+$ npm install
+$ npm test ../config.splitting.js
+$ npm run test:html # to test config.splitting.js against real HTML files
```

Comment:
`find: **/*.html: No such file or directory`

Reply:
=> I added explanations in the README